class-gwiz-gf-feed-forge.php

```diff
@@ -243,6 +243,7 @@ public function localize_admin_script( $form_id ) {
 			'GFFF_ADMIN',
 			[
 				'nonce'             => wp_create_nonce( 'gf_process_feeds' ),
+				'processor_nonce'   => wp_create_nonce( 'wp_gf_feed_processor' ), // Generate nonce for feed processor to pass to client
```

Comment:
@saifsultanc can you leave a note here that this exact `action` value is required here?

Reply:
Done!